translations/ca.json

```diff
@@ -2164,6 +2164,7 @@
                             "name": "Filtre d'entitats"
                         },
                         "entity": {
+                            "description": "La targeta entitat mostra una visualització ràpida dels estats d'una o més entitats.",
```

Comment:
All translations will need removed except for `src/translations/en.json`

Reply:
I believe I fixed.

Reply:
Good now